map/framework.cpp

```diff
@@ -2319,6 +2319,27 @@ bool Framework::ParseEditorDebugCommand(search::SearchParams const & params)
   return false;
 }
 
+namespace
+{
+vector<string> FilterNearbyStreets(vector<search::ReverseGeocoder::Street> const & streets)
```

Comment:
I am asking you to do in-place filtering, without creation of a new vector. Why not to pass argument by reference and remove unnecessary elements from it?